src/views/docs-view/server.ts

```diff
@@ -46,13 +46,14 @@ import { getCustomLayout } from './utils/get-custom-layout'
  * will no longer be necessary, and we will no longer pass the
  * "latestVersionRef" to the remote content loader config.
  */
-function getBetaLatestVersionRef(slug: string): string | undefined {
+export function getBetaLatestVersionRef(slug: string): string | undefined {
 	const hasDevPortalBranch = [
 		'vault',
 		'waypoint',
 		'boundary',
 		'consul',
 		'nomad',
+		'packer',
```

Comment:
Have added this to signal that Packer has a `dev-portal` branch which we'll pull from.